source/common/upstream/health_discovery_service.cc

```diff
@@ -393,7 +394,7 @@ void HdsCluster::update(envoy::config::cluster::v3::Cluster cluster,
     if (socket_match_hash_ != socket_match_hash) {
       socket_match_hash_ = socket_match_hash;
       update_cluster_info = true;
-      info_ = info_factory.createClusterInfo({server_context_, cluster_, bind_config_, stats_,
+      info_ = info_factory.createClusterInfo({server_context_, cluster_, bind_config, stats_,
```

Comment:
with this change, we no longer need `bind_config_` member in HdsCluster?

Reply:
Yes, you are right, they are defined in `Cluster`. I'll change it